src/test/test_pidfile.sh

```diff
@@ -50,11 +50,11 @@ function TEST_pidfile() {
 
     # no daemon can use a pidfile that is owned by another daemon
     run_mon $dir a || return 1
-    sleep 2
+    sleep 5
```

Comment:
We saw this test fail, again.  We are starting a background daemon then trying to start another instance.  The first invocation must make enough progress for it to lock its pid file.